src/renderer/src/views/Home/index.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+
+import styled from '@emotion/styled'
+import { theme } from '@renderer/theme'
+
+import { Sidebar } from './Sidebar'
+import { TabPanel } from './TabPanel'
+import { DefaultLayout } from '@renderer/layouts/default'
+
+const Container = styled.div(`
+  flex: 1;
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-template-rows: 1fr auto;
+`)
+
+const SidebarGridSection = styled.div(`
+  grid-area: 1 / 1 / 2 / 2;
+  display: flex;
+  flex: 1;
+`)
+
+const MainGridSection = styled.div(`
+  grid-area: 1 / 2 / 2 / 3;
+  position: relative;
+`)
+
+const BottomBarGridSection = styled.div(`
+  grid-area: 2 / 1 / 3 / 3;
+  background: ${theme.blueDark};
+`)
+
+export const PANEL_NAMES = {
+  territory: 'territory',
+  observations: 'observations',
+  sync: 'sync',
+  settings: 'settings',
+} as const
```

Comment:
this const is unnecessary. You can just create a type with all the PanelNames

Reply:
I'm using this as an enum-like pattern (see Sidebar), although just noticed that I'm not using it in this component when I should be 😅 either way, can update to remove since not super necessary

Reply:
addressed via https://github.com/digidem/mapeo-desktop-shell/pull/23/commits/4f94e94ac9c66ab7d0b27a5318c14fe92e3db3a7